README.md

```diff
@@ -40,131 +40,152 @@ FIEDLER, Niklas, et al. [ImageTagger: An Open Source Online Platform for Collabo
    organization={Springer}
 }
 ```
-## Upgrade
 
-```
-pip install -U -r requirements.txt
-./manage.py migrate
-```
-
-for additional steps on some releases see instructions
-in [UPGRADE.md](https://github.com/bit-bots/imagetagger/blob/master/UPGRADE.md)
-
-## Install
-
-Checkout the latest release:
-
-```
-git checkout v0.x
-```
-
-In our production Senty is used for error reporting (pip install sentry-sdk).
-django-auth-ldap is used for login via ldap
-uwsgi is used to serve the app to nginx
-
-Install Python Dependencies:
-
-```
-pip3 install -r requirements.txt
-```
-
-Copy settings.py.example to settings.py in the imagetagger folder:
-
-```
-cp imagetagger/settings.py.example imagetagger/settings.py
-```
-
-and customize the settings.py.
-
-The following settings should probably be changed:
-
-+ The secret key
-+ The DEBUG setting
-+ The ALLOWED\_HOSTS
-+ The database settings
```

Comment:
I think that this information is also very relevant to a user who just wants to quickly setup the ImageTagger.

Reply:
I have readded it in 696b89fcedc12ce9b3a588363b8c112b9dcd98bf